examples/parcel/.parcelrc

```diff
@@ -1,9 +1,14 @@
 {
   "extends": "@parcel/config-default",
   "transformers": {
-    "*.{js,jsx,ts,tsx}": [
+    "*.{js,jsx,ts,tsx,customjsx}": [
       "@compiled/parcel-transformer",
       "..."
+    ],
+    "*.customjsx": [
```

Comment:
Is this one needed?

Reply:
Yep, we get a runtime error from the missing file without it